download-utils/src/lib.rs

```diff
@@ -108,17 +110,79 @@ pub fn download_file(url: &str, destination_file: &Path) -> Result<(), String> {
     Ok(())
 }
 
-fn extract_archive(archive_filename: &Path, destination_dir: &Path) -> Result<(), String> {
+const MAXIMUM_GENESIS_ARCHIVE_SIZE: u64 = 10 * 1024 * 1024; // 10 MiB
```

Comment:
This is the uncompressed size right?  Can this be indicated in the const name somehow please.